lib/routes/home_route/tabs_route.dart

```diff
@@ -192,13 +192,7 @@ class TabsRouteState extends State<TabsRoute> with TickerProviderStateMixin, Sel
         drawerController.open();
       },
       actions: selectionRoute ? const [] : [
-        Visibility(
-          visible: false,
-          maintainState: true,
-          maintainAnimation: true,
-          maintainSize: true,
-          child: DeleteSongsAppBarAction<Content>(controller: selectionController),
-        ),
+        const _ShowOnlyFavoritesButton(),
```

Comment:
This conceptually contradicts the context of this header, because it acts globally, entering favorite filter for all content, while all other elements of this header are related to a specific content (e.g. Song on your screenshot)